src/In.ProjectEKA.HipService/appsettings.json

```diff
@@ -43,5 +43,9 @@
     "url": "https://ncg-dev.projecteka.in/central-registry",
     "clientId": "10000005",
     "clientSecret": "93ef283e-1968-46ab-8731-cfccda83680a"
+  },
+  "Gateway": {
+    "url": "http://localhost:9003",
+    "timeout": 3
```

Comment:
are you sure that it's localhost? can you check with @Apoorva-GA / @ajayvigneshk  what is the URL in dev you need to use

Reply:
Got updated by mistake, it should be http://gateway:8000 - have made the change